src/app/features/admin/components/carriage-preview/carriage-preview.component.scss

```diff
@@ -0,0 +1,87 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+header {
+  color: var(--tui-text-secondary);
+}
+
+section {
+  overflow: auto;
+  align-items: center;
+  justify-content: center;
+
+  color: var(--tui-text-secondary);
+
+  border: 1px dashed;
+  border-radius: 0.75rem;
+}
+
+.carriage {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  align-items: center;
+  justify-content: center;
+
+  padding: 1rem;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  align-items: center;
+  justify-content: center;
+}
+
+.seats-right,
+.seats-left {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.seat {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: 20px;
+  height: 20px;
+
+  font-size: 12px;
+  color: var(--tui-background-base-alt);
+  text-align: center;
+
+  background-color: var(--tui-background-accent-1-pressed);
+  border-radius: 5px;
+
+  &::before {
+    content: '';
+
+    position: absolute;
+    top: -2px;
+    left: -2px;
+
+    width: 10px;
+    height: 23px;
+
+    border-top: 1px solid var(--tui-status-neutralt);
+    border-bottom: 1px solid var(--tui-status-neutral);
+    border-left: 1px solid var(--tui-status-neutral);
+    border-top-left-radius: 5px;
+    border-bottom-left-radius: 5px;
+  }
+}
+
+.aisle {
+  height: 30px;
+  background-color: #f3f4f6;
+}
+
+@media #{$tui-mobile} {
+  .carriage {
+    justify-content: flex-start;
+  }
+}
```

Comment:
![image](https://github.com/user-attachments/assets/aee2c40f-f68c-4f38-9516-832670446aef)
we can add this styles to achieve this effect on mobile devices:

![image](https://github.com/user-attachments/assets/224f4b93-674f-4e64-932d-295d6e973a02)

Reply:
I accidentally approved